client/components/LearnerViewDirective.js

```diff
@@ -773,7 +801,15 @@ tie.directive('learnerView', [function() {
             for (var i = 0; i < feedbackParagraphs.length; i++) {
               if (feedbackParagraphs[i].isSyntaxErrorParagraph()) {
                 hasSyntaxError = true;
+                clearHighlight();
```

Comment:
There's a bit more refactoring that's needed here, I think. Syntax errors are no longer feedback paragraphs, so they should really be a separate construct altogether. Maybe we should chat about this...